backend/app/treasury/actions.py

```diff
@@ -44,7 +44,7 @@ def fill_asset_hist_balance(
     ) -> Optional[pd.DataFrame]:
         if (
             symbol not in asset_transfer_balances
-            or len(asset_transfer_balances[symbol]) < 2
+            or len(asset_transfer_balances[symbol]) == 0
```

Comment:
Same as above: the transfer list is never empty, as it's already filtered in `_get_asset_transfer_balances`.